examples/custom_role_org/main.tf

```diff
@@ -41,3 +41,12 @@ module "custom-roles-org" {
   permissions  = ["iam.roles.list", "iam.roles.delete"]
   description  = "This is an organization level custom role."
 }
+
+/******************************************
+  Assigning custom_role to members
+ *****************************************/
+resource "google_organization_iam_binding" "custom_role_members" {
```

Comment:
Please use the `iam_member` resource, not `iam_binding`. iam_binding is destructive/dangerous.

Reply:
Initially I had used iam_binding to add multiple members at the same time. But I see your point. Changed to the iam_member resource.